packages/@react-spectrum/dropzone/stories/DropZone.stories.tsx

```diff
@@ -0,0 +1,259 @@
+          setIsFilled(!!items);
+          setFilledSrc(items.join('\n'));
+        }}
+        onDropEnter={action('onDropEnter')}
+        onDropExit={action('onDropExit')} 
+        onPaste={action('onPaste')}>
+        <IllustratedMessage>
+          <Upload />
+          <Heading>Drag and Drop here</Heading>
+        </IllustratedMessage>
+        {isFilled && <div style={{width: '100px', position: 'absolute'}}>{filledSrc}</div>}
+      </DropZone>
+    </>
+  );
+}
+
+function DropZoneWithLink(props) {
+  let [isFilled, setIsFilled] = useState(false);
+  let [filledSrc, setFilledSrc] = useState(null);
+
+  return (
+    <DropZone 
+      {...props}
+      isFilled={isFilled}
+      onDrop={async (e) => { 
+        let item = e.items.find((item) => item.kind === 'file') as FileDropItem;
+        if (item) {
+          setFilledSrc(URL.createObjectURL(await item.getFile()));
+        }
+      }}
+      onDropEnter={action('onDropEnter')}
+      onDropExit={action('onDropExit')} 
+      onPaste={action('onPaste')}>
+      <IllustratedMessage>
+        <Upload />
+        <Heading>Drag and Drop here</Heading>
+        <Content>
+          <FileTrigger
+            onChange={(e) => {
+              let files = Array.from(e);
+              let urls = files.map(file => URL.createObjectURL(file));
+              setFilledSrc(urls[0]);
+              setIsFilled(true);
+            }}>
+            <Link>Select a file</Link> from your computer
+          </FileTrigger>
+        </Content>
+      </IllustratedMessage>
+      {isFilled && <img style={{width: '100px', position: 'absolute'}} src={filledSrc} alt="test" />}
+    </DropZone>
+  );
+}
+
+function DropZoneWithButton(props) {
+  let [isFilled, setIsFilled] = useState(false);
+  let [filledSrc, setFilledSrc] = useState(null);
+
+  return (
+    <DropZone 
+      {...props}
+      isFilled={isFilled}
+      onDrop={async (e) => { 
+        let item = e.items.find((item) => item.kind === 'file') as FileDropItem;
+        if (item) {
+          setFilledSrc(URL.createObjectURL(await item.getFile()));
+        }
+      }}
+      onDropEnter={action('onDropEnter')}
+      onDropExit={action('onDropExit')} 
+      onPaste={action('onPaste')}>
+      <IllustratedMessage>
+        <Upload />
+        <Heading>Drag and Drop here</Heading>
+        <Content>
+          <FileTrigger
+            onChange={(e) => {
+              let files = Array.from(e);
+              let urls = files.map(file => URL.createObjectURL(file));
+              setFilledSrc(urls[0]);
+              setIsFilled(true);
+            }}>
+            <Button variant={'accent'}>Browse</Button>
+          </FileTrigger>
+        </Content>
+      </IllustratedMessage>
+      {isFilled && <img style={{width: '100px', position: 'absolute'}} src={filledSrc} alt="test" />}
+    </DropZone>
+  );
+}
+
+function Draggable() {
```

Comment:
Might be easier exporting and importing this from the aria DnD stories like its done for Droppable in ListViewDnD stories